pkg/apis/triggers/v1alpha1/event_listener_validation.go

```diff
@@ -67,7 +67,7 @@ func (s *EventListenerSpec) validate(ctx context.Context, el *EventListener) *ap
 
 func (i *EventInterceptor) validate(ctx context.Context, namespace string) *apis.FieldError {
 	// Validate at least one
-	if i.Webhook == nil && i.Github == nil && i.Gitlab == nil {
+	if i.Webhook == nil && i.Github == nil && i.Gitlab == nil && i.CEL == nil {
```

Comment:
Should we also validate that if CEL has the `expression` field set?

Reply:
Add validation for the renamed `filter` expression.